src/main/java/me/crespel/runtastic/converter/ExportConverter.java

```diff
@@ -11,41 +11,84 @@
 
 import me.crespel.runtastic.mapper.DelegatingSportSessionMapper;
 import me.crespel.runtastic.mapper.SportSessionMapper;
+import me.crespel.runtastic.model.ImagesMetaData;
 import me.crespel.runtastic.model.SportSession;
+import me.crespel.runtastic.model.SportSessionAlbums;
+import me.crespel.runtastic.model.User;
 import me.crespel.runtastic.parser.SportSessionParser;
 
 /**
  * Export directory converter.
  * @author Fabien CRESPEL (fabien@crespel.net)
+ * @author Christian IMFELD (imfeldc@gmail.com)
  */
 public class ExportConverter {
 
 	public static final String SPORT_SESSIONS_DIR = "Sport-sessions";
+	public static final String PHOTOS_DIR = "Photos";
+	public static final String PHOTOS_META_DATA_DIR = "Photos\\Images-meta-data";
+	public static final String PHOTOS_SPORT_SESSION_ALBUMS_DIR = "Photos\\Images-meta-data\\Sport-session-albums";

```

Comment:
Don't use Windows separators (`\\`), prefer adding `File.separator`.
Also, there is some code duplication with SportSessionParser, consider moving all static constants to a dedicated ExportConstants class or interface.

Reply:
Thanks! Use File.separator. Will be part of next delivery.

Reply:
Open https://github.com/fcrespel/runtastic-export-converter/issues/5 to track further changes.